MapboxDirections/Match/MBMatchOptions.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+@objc(MBMatchOptions)
+open class MatchOptions: RouteOptions {
+    public init(tracePoints: [TracePoint], profileIdentifier: MBDirectionsProfileIdentifier?, resample: Bool?, timestamps: [Date]?) {
+        super.init(waypoints: tracePoints, profileIdentifier: profileIdentifier)
+    }
+    
+    @objc open var resample: Bool = false
```

Comment:
“Resample” sounds like a method name. Rename this to `resamples<some noun>` or `tidies<some noun>`.